packages/sodium/lib/src/js/sodium_js_init.dart

```diff
@@ -17,9 +17,10 @@ abstract class SodiumInit {
   /// The [libsodium] parameter must be a loaded
   /// `[lib]sodium.[so|dll|dylib|a|lib|js]`- depending on your platform. Please
   /// refer to the README for more details on loading the library.
-  ///
-  /// The [initNative] parameter has no effect on the web.
-  static Future<Sodium> init(dynamic libsodium, {bool initNative = true}) =>
+  static Future<Sodium> init(
+          dynamic libsodium,
+          {@Deprecated('initNative is deprecated and will be ignored.')
+              bool initNative = true}) =>
```

Comment:
Please add a ',' after init native for nicer formatting with `dart format`

Reply:
Great idea, fixed that up.